dom/exercise6-calculator.js

```diff
@@ -0,0 +1,68 @@
+// # Create a calculator
+// # This seems simple but we are going to make it just like a traditional
+// # calculator where the user can enter any values and any operand
+// # Example: 
+//     # input: 5 + 10 - 2 / 15 * 20 + 99999
+//     # output: the result of above
+// # make sure to handle any data type such as int and floats
+// # make sure to apply the correct mathmetical sequence to process the equation
+
+// get text into the display box
+let displayBox = document.querySelector("#display-box");
+
+function displayValue(x) {
+    if (x !== "Enter") {
+        displayBox.textContent += x;
+        return x;
+    };
+}
+
+// clear display when needed
+function clearDisplay() {
+    displayBox.textContent = "";
+}
+
+// evaluate the text in the display box and complete the equation
+function doTheMath() {
+    let equation = displayBox.textContent;
+    let finalize = eval(equation);
```

Comment:
Please remove eval unless you do some ***heavy*** sanitization.

Reply:
provided new resolution instead of eval()